webrender/src/device/gfx.rs

```diff
@@ -50,7 +50,8 @@ pub const INVALID_TEXTURE_ID: TextureId = 0;
 pub const INVALID_PROGRAM_ID: ProgramId = ProgramId(0);
 pub const DEFAULT_READ_FBO: FBOId = FBOId(0);
 pub const DEFAULT_DRAW_FBO: FBOId = FBOId(1);
-pub const MAX_FRAME_COUNT: usize = 2;
+const MAX_FRAME_COUNT: usize = 2;
+const DESCRIPTOR_COUNT: usize = 400;
```

Comment:
IIRC for dx12, this will be too much. But please try it and we will know for sure.